### PR TITLE
Change how Gorelease strips internal changelog messages

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: provider-ci/lib/goreleaser.js [hunk 1]
```diff
@@ -87,9 +87,9 @@ export class PulumiGoreleaserConfig extends PulumiGoreleaserPreConfig {
                 exclude: [
                     "Merge branch",
                     "Merge pull request",
-                    "[internal]",
-                    "[ci]",
-                    "[chore]",
+                    "internal",
```
Comment (guineveresaenger) [inline provider-ci/lib/goreleaser.js:90]: should we keep the square brackets as part of the pattern though? we don't want to accidentally exclude any commits that just have this word in them?

File: provider-ci/providers/aiven/repo/.goreleaser.yml [hunk 2]
```diff
@@ -33,9 +33,9 @@ changelog:
     exclude:
     - Merge branch
     - Merge pull request
-    - '[internal]'
-    - '[ci]'
-    - '[chore]'
+    - internal
```
Comment (guineveresaenger) [inline provider-ci/providers/aiven/repo/.goreleaser.yml:36]: I think we should come up with a way to escape the square brackets so they're part of the regex.

Comment (guineveresaenger) [inline provider-ci/providers/aiven/repo/.goreleaser.yml:36]: I think we should come up with a way to generate this to say `[internal]`.

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
I think this is going to improve things but I have a concern that it's going to catch real commit messages accidentally.

File: provider-ci/providers/aiven/repo/.goreleaser.yml [hunk 3]
```diff
@@ -33,9 +33,9 @@ changelog:
     exclude:
     - Merge branch
     - Merge pull request
-    - '[internal]'
-    - '[ci]'
-    - '[chore]'
+    - WinternalW
```
Comment (guineveresaenger) [inline provider-ci/providers/aiven/repo/.goreleaser.yml:36]: This doesn't look right either.. we want it to still say `\Wchore\W` no?

Comment (stack72) [reply provider-ci/providers/aiven/repo/.goreleaser.yml:36]: The \ isn’t being kept in the generate… that’s weird

**Review by jkodroff — APPROVED** [review]

**Review by guineveresaenger — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**